src/renderer/vt/paint.cpp

```diff
@@ -448,10 +448,13 @@ using namespace Microsoft::Console::Types;
     // the lines _wrapped_. It doesn't care to manually break the lines, but if
     // we trimmed the spaces off here, we'd print all the "~"s one after another
     // on the same line.
-    const auto removeSpaces = !lineWrapped && (useEraseChar ||
-                                               _clearedAllThisFrame ||
-                                               (_newBottomLine && printingBottomLine && bgMatched));
-    const auto cchActual = removeSpaces ? nonSpaceLength : cchLine;
+    static const TextAttribute defaultAttrs{};

```

Comment:
It has been so long that I can't recall whether I wrote this

Reply:
you had it inline:

* you: aba6caf
* my delta: 1fdd632